bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsMagnitudeTable.scala

```diff
@@ -115,19 +149,24 @@ object GemsMagnitudeTable extends MagnitudeTable {
    * be used directly by Mascot, since it cannot be looked up through the GemsInstrumentToMagnitudeLimitsCalculator map.
    */
   trait CanopusWfsCalculator extends LimitsCalculator {
-    def getNominalMagnitudeConstraints(cwfs: Canopus.Wfs): MagnitudeConstraints
+    def getNominalMagnitudeConstraints(cwfs: CanopusWfs): MagnitudeConstraints
```

Comment:
It's not new in the PR, but up above (and unchanged from previous versions) the `searchCriterionBuilder` picks one position angle at random (well, one from a possibly unordered `Set`).  The pos angle matters because the guide star search center for GSAOI is offset by a bit from the center. So anyway, this seems sketchy.

Reply:
Perhaps the instrument offset should just appropriately embiggen the search radius provided to the catalog to incorporate any position angle in the candidate set.